project/ScalafixBuild.scala

```diff
@@ -175,10 +175,12 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
   )
 
   override def buildSettings: Seq[Setting[_]] = List(
+    // https://github.com/sbt/sbt/issues/5568#issuecomment-1094380636
+    versionPolicyIgnored += "com.lihaoyi" %% "pprint",
     versionPolicyIgnoredInternalDependencyVersions :=
       Some("^\\d+\\.\\d+\\.\\d+\\+\\d+".r),
     versionScheme := Some("early-semver"),
-    versionPolicyIntention := Compatibility.None // TODO: harden after 0.10.0
+    versionPolicyIntention := Compatibility.BinaryCompatible
```

Comment:
since we are still in 0.x.y, we'll need to bump a minor anyway for the next release (and not a "semantic" patch), so there is not interest in checking `BinaryAndSourceCompatible`